docs/atomic-structure.md

```diff
@@ -142,7 +142,7 @@ atomicHelpers.instantiateAll( '.o-expandable', Expandable );
 
 ### Templates
 
-Prefixed with “t-” in CSS, JavaScript, and HTML.
+Prefixed with “t-” in CSS, JavaScript, and HTML. [View all available templates](https://github.com/cfpb/cfgov-refresh/tree/master/cfgov/jinja2/v1) that can be extended or reused to create pages.
```

Comment:
This prefixing is not something we actually do. Should we remove the reference to that?

Reply:
We've never really followed the atomic template principals, but maybe we should to stay consistent?

Reply:
It was used for a time in `.t-careers`.